README.md

```diff
@@ -2,11 +2,25 @@
 
 Terraform module which creates ECS (Elastic Container Service) resources on AWS.
 
+[![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md)
+
 ## Available Features
 
 - ECS cluster
 - Fargate capacity providers
 - EC2 AutoScaling Group capacity providers
+- ECS Service sub-module
```

Comment:
Maybe reduce the number of details shown here next to each feature. It is already described in README of the submodule and examples.

Reply:
done - moved a lot of this stuff to a design doc